libs/openFrameworks/graphics/ofPixels.h

```diff
@@ -1368,7 +1371,7 @@ inline typename ofPixels_<PixelType>::ConstPixel ofPixels_<PixelType>::ConstPixe
 
 //----------------------------------------------------------------------
 template<typename PixelType>
-inline ofPixels_<PixelType>::ConstLine::ConstLine(const PixelType * _begin, size_t stride, size_t componentsPerPixel, size_t lineNum, ofPixelFormat pixelFormat)
+inline ofPixels_<PixelType>::ConstLine::ConstLine(const PixelType * _begin, size_t stride, size_t componentsPerPixel, size_t lineNum, ofPixelFormat _pixelFormat)
```

Comment:
why the _

Reply:
to differentiate the function parameter to the private member called pixelFormat;